# A S Alam <aalam@fedoraproject.org>, 2017. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-06-13 09:18-0400\n"
"PO-Revision-Date: 2017-08-18 07:13+0000\n"
"Last-Translator: A S Alam <aalam@fedoraproject.org>\n"
"Language-Team: Punjabi\n"
"Language: pa\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"X-Generator: Zanata 4.6.2\n"

#: plugins/kickstart.py:57
msgid "Install packages defined in a kickstart file on your system"
msgstr ""

#: plugins/kickstart.py:62
msgid "kickstart file"
msgstr "ਕਿੱਕਸਟਾਰਟ ਫਾਇਲ"

#: plugins/kickstart.py:80
#, python-format
msgid "file cannot be parsed: %s"
msgstr "ਫਾਇਲ ਨੂੰ ਪਾਰਸ ਨਹੀਂ ਕੀਤਾ ਜਾ ਸਕਦਾ: %s"

#: plugins/rpm_conf.py:65
msgid "Adding '{}' to list of handling packages for rpmconf"
msgstr ""

#: plugins/rpm_conf.py:71
msgid "rpmconf plugin will not run in non-interactive mode"
msgstr ""

#: plugins/rpm_conf.py:84
msgid "ignoring sys.exit from rpmconf due to missing MERGE variable"
msgstr ""

#: plugins/rpm_conf.py:88
msgid "ignoring sys.exit from rpmconf due to missing file"
msgstr ""

#: plugins/showvars.py:46
msgid "show all active dnf variables"
msgstr ""

#: plugins/snapper.py:49
#, python-format
msgid "connect to snapperd failed: %s"
msgstr ""

#: plugins/snapper.py:55
msgid "creating pre_snapshot"
msgstr ""

#: plugins/snapper.py:61
#, python-format
msgid "created pre_snapshot %d"
msgstr ""

#: plugins/snapper.py:65
#, python-format
msgid "creating pre_snapshot failed: %s"
msgstr ""

#: plugins/snapper.py:74
msgid "skipping post_snapshot because creation of pre_snapshot failed"
msgstr ""

#: plugins/snapper.py:80
msgid "creating post_snapshot"
msgstr ""

#: plugins/snapper.py:85
#, python-format
msgid "created post_snapshot %d"
msgstr ""

#: plugins/snapper.py:89
#, python-format
msgid "creating post_snapshot failed: %s"
msgstr ""

#. Translators: This string is only used in unit tests.
#: plugins/system_upgrade.py:45
msgid "the color of the sky"
msgstr "ਅਸਮਾਨ ਦਾ ਰੰਗ"

#: plugins/system_upgrade.py:57
msgid "Need a --releasever greater than the current system version."
msgstr ""

#. Translators: do not change "reboot" here
#: plugins/system_upgrade.py:59
#, python-brace-format
msgid ""
"Download complete! Use 'dnf {command} reboot' to start the upgrade.\n"
"To remove cached metadata and transaction use 'dnf {command} clean'"
msgstr ""

#: plugins/system_upgrade.py:62
msgid "Sorry, you need to use 'download --releasever' instead of '--network'"
msgstr ""

#: plugins/system_upgrade.py:71
msgid "Reboot turned off, not rebooting."
msgstr ""

#: plugins/system_upgrade.py:120
#, python-format
msgid "Screen blanking can't be disabled: %s"
msgstr ""

#: plugins/system_upgrade.py:140
#, python-format
msgid "Failed loading state file: %s, continuing with empty state."
msgstr ""

#: plugins/system_upgrade.py:286
msgid "The following boots appear to contain upgrade logs:"
msgstr ""

#: plugins/system_upgrade.py:296
msgid "-- no logs were found --"
msgstr "-- ਕੋਈ ਲਾਗ ਨਹੀਂ ਲੱਭਿਆ --"

#: plugins/system_upgrade.py:311
msgid "Cannot find logs with this index."
msgstr ""

#: plugins/system_upgrade.py:320
msgid "Unable to match systemd journal entry"
msgstr ""

#: plugins/system_upgrade.py:341
msgid "Prepare system for upgrade to a new release"
msgstr "ਸਿਸਟਮ ਨੂੰ ਨਵੇਂ ਰੀਲਿਜ਼ ਵਾਸਤੇ ਅੱਪਗਰੇਡ ਲਈ ਤਿਆਰ ਕੀਤਾ ਜਾ ਰਿਹਾ ਹੈ"

#: plugins/system_upgrade.py:357
msgid "keep installed packages if the new release's version is older"
msgstr ""

#: plugins/system_upgrade.py:361
msgid "which logs to show"
msgstr ""

#: plugins/system_upgrade.py:395
#, python-brace-format
msgid "Incompatible version of data. Rerun 'dnf {command} download [OPTIONS]'"
msgstr ""

#: plugins/system_upgrade.py:432
msgid "Command 'offline-distrosync' cannot be used with --no-downgrade option"
msgstr ""

#: plugins/system_upgrade.py:454
msgid "Additional information for System Upgrade: {}"
msgstr ""

#: plugins/system_upgrade.py:457
msgid ""
"Before you continue ensure that your system is fully upgraded by running "
"\"dnf --refresh upgrade\". Do you want to continue"
msgstr ""

#: plugins/system_upgrade.py:461
msgid "Operation aborted."
msgstr ""

#: plugins/system_upgrade.py:519
msgid "system is not ready for upgrade"
msgstr "ਸਿਸਟਮ ਅੱਪਗਰੇਡ ਕਰਨ ਲਈ ਤਿਆਰ ਨਹੀਂ ਹੈ"

#: plugins/system_upgrade.py:522
#, python-brace-format
msgid ""
"the transaction was not prepared for '{command}'. Rerun 'dnf {command} "
"download [OPTIONS]'"
msgstr ""

#: plugins/system_upgrade.py:526
msgid "upgrade is already scheduled"
msgstr "ਅੱਪਗਰੇਡ ਲਈ ਪਹਿਲਾਂ ਹੀ ਸਮਾਂ ਤਹਿ ਕੀਤਾ ਗਿਆ"

#: plugins/system_upgrade.py:532
msgid "trigger file does not exist. exiting quietly."
msgstr ""

#: plugins/system_upgrade.py:535
msgid "another upgrade tool is running. exiting quietly."
msgstr "ਹੋਰ ਅੱਪਗਰੇਡ ਟੂਲ ਚੱਲ ਰਿਹਾ ਹੈ। ਬੰਦ ਕੀਤਾ ਜਾ ਰਿਹਾ ਹੈ।"

#: plugins/system_upgrade.py:544
#, python-brace-format
msgid "use 'dnf {command} reboot' to begin the upgrade"
msgstr ""

#: plugins/system_upgrade.py:562
msgid "Rebooting to perform upgrade."
msgstr "ਅੱਪਗਰੇਡ ਕਰਨ ਲਈ ਮੁੜ-ਚਾਲੂ ਕੀਤਾ ਜਾ ਰਿਹਾ ਹੈ।"

#: plugins/system_upgrade.py:595
msgid "Starting offline upgrade. This will take a while."
msgstr ""

#: plugins/system_upgrade.py:597
msgid "Starting offline distrosync. This will take a while."
msgstr ""

#: plugins/system_upgrade.py:599
msgid "Starting system upgrade. This will take a while."
msgstr "ਸਿਸਟਮ ਨੂੰ ਅੱਪਗਰੇਡ ਕਰਨਾ ਸ਼ੁਰੂ ਕੀਤਾ ਜਾ ਰਿਹਾ ਹੈ। ਇਸ ਨੂੰ ਕੁਝ ਸਮਾਂ ਲੱਗੇਗਾ।"

#: plugins/system_upgrade.py:615
msgid "Cleaning up downloaded data..."
msgstr "...ਡਾਊਨਲੋਡ ਕੀਤੇ ਡਾਟੇ ਨੂੰ ਸਾਫ਼ ਕੀਤਾ ਜਾ ਰਿਹਾ ਹੈ"

#: plugins/system_upgrade.py:644
msgid ""
"The system-upgrade transaction is empty, your system is already up-to-date."
msgstr ""

#: plugins/system_upgrade.py:653
msgid "Transaction saved to {}."
msgstr ""

#: plugins/system_upgrade.py:656
msgid "Error storing transaction: {}"
msgstr ""

#: plugins/system_upgrade.py:678
msgid "Download finished."
msgstr "ਡਾਊਨਲੋਡ ਕਰਨਾ ਪੂਰਾ ਹੋਇਆ।"

#: plugins/system_upgrade.py:681 plugins/system_upgrade.py:682
msgid "Upgrade complete! Cleaning up and rebooting..."
msgstr "ਅੱਪਗਰੇਡ ਪੂਰਾ ਹੋਇਆ! ਸਾਫ਼ ਕਰਕੇ ਮੁੜ-ਚਾਲੂ ਕੀਤਾ ਜਾ ਰਿਹਾ ਹੈ..."

#: plugins/system_upgrade.py:691
msgid "Prepare offline upgrade of the system"
msgstr ""

#: plugins/system_upgrade.py:696
msgid "Prepare offline distrosync of the system"
msgstr ""

#: plugins/torproxy.py:31
msgid "Disabling torproxy plugin: cannot connect to the Tor network"
msgstr ""

#: plugins/tracer.py:69
msgid "For more information run:"
msgstr "ਹੋਰ ਜਾਣਕਾਰੀ ਲਈ ਚਲਾਓ:"

#: plugins/tracer.py:96
msgid "Call to Tracer API ended unexpectedly:"
msgstr ""

#: plugins/tracer.py:98
msgid ""
"Please visit https://github.com/FrostyX/tracer/issues and submit the issue. "
"Thank you"
msgstr ""

#: plugins/tracer.py:100
msgid "We apologize for any inconvenience"
msgstr "ਕਿਸੇ ਵੀ ਆਈ ਸਮੱਸਿਆ ਲਈ ਸਾਨੂੰ ਅਫ਼ਸੋਸ ਹੈ"
